src/components/list-items/list-items.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+import './list-items.css';
+
+export default class ListItems extends Component {
+  render() {
+    const { usersId } = this.props
+    return (
+      <div className='list-items'>
+        <div className='check-box'>
```

Comment:
`check-box` can be renamed to `user-filters` class, as basically it's a parent div for all filters